HACKING.md

```diff
@@ -136,6 +136,21 @@ opam repo set-url pin git+https://github.com/ocaml/opam-repository#<commit-hash>
 
 Where `<commit-hash>` is the pinned hash specified in the files mentioned above.
 
+
+## FAQ
+
+### How to Handle Warnings
+
+1. Outdated Browserslist Warning:
+   - To fix an outdated Browserslist warning, run:
+     ```bash
+     npx update-browserslist-db@latest
+     ```
```

Comment:
When I then try to run ` npx update-browserslist-db@latest` I get this:

```
$ npx update-browserslist-db@latest
Need to install the following packages:
  update-browserslist-db@latest
Ok to proceed? (y) 
update-browserslist-db: Cannot find package.json. Is this the right directory to run `npx update-browserslist-db` in?
```

---

Here is what the warning message says when I run `make`:

```
$ make
opam exec -- dune build --root .
Browserslist: caniuse-lite is outdated. Please run:
  npx browserslist@latest --update-db
  Why you should do it regularly: https://github.com/browserslist/browserslist#browsers-data-updating

Done in 1574ms.
```


When I run what the warning message suggests me to, I get this:

```
$ npx browserslist@latest --update-db
Need to install the following packages:
  browserslist@latest
Ok to proceed? (y) y
The --update-db command is deprecated.
Please use npx update-browserslist-db@latest instead.
Browserslist DB update will still be made.

/home/sabine-tarides/.npm/_npx/072487b38308305a/node_modules/update-browserslist-db/index.js:31
    throw new BrowserslistUpdateError(
    ^
Error [BrowserslistUpdateError]: Cannot find package.json. Is this the right directory to run `npx update-browserslist-db` in?
    at detectLockfile (/home/sabine-tarides/.npm/_npx/072487b38308305a/node_modules/update-browserslist-db/index.js:31:11)
    at updateDB (/home/sabine-tarides/.npm/_npx/072487b38308305a/node_modules/update-browserslist-db/index.js:271:14)
    at Object.<anonymous> (/home/sabine-tarides/.npm/_npx/072487b38308305a/node_modules/browserslist/cli.js:47:3)
    at Module._compile (internal/modules/cjs/loader.js:999:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1027:10)
    at Module.load (internal/modules/cjs/loader.js:863:32)
    at Function.Module._load (internal/modules/cjs/loader.js:708:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:60:12)
    at internal/main/run_main_module.js:17:47 {
  browserslist: true
}
```

So the suggestion to run `npx update-browserslist-db@latest` seems correct, but it doesn't work on my machine for some reason. Does it work on yours?

Reply:
Yes it works on my machine

Reply:
> Yes it works on my machine

![Screenshot from 2023-10-14 15-58-08](https://github.com/ocaml/ocaml.org/assets/48816919/88f4bdde-9bc1-4c7d-83bd-82416e42c665)

Reply:
> Yes it works on my machine

![Screenshot from 2023-10-14 16-03-47](https://github.com/ocaml/ocaml.org/assets/48816919/5807e5dd-037e-4b8c-ad0a-11f1ebebf6ae)